src/__screenshot_tests__/input-fields-screenshot-test.tsx

```diff
@@ -498,6 +498,13 @@ test.each`
     const field = await screen.findByLabelText('Label');
 
     await field.click({clickCount: 3});
+
```

Comment:
Unrelated. This test seems to be unstable, I've created a task to review it: https://jira.tid.es/browse/WEB-2047